src/index.ts

```diff
@@ -353,7 +353,7 @@ class Envoy {
         log.normal("Completed sign-in process for Agent (%s) HHA ID (%s)", agent_id, hha_hash);
         return res;
       } catch (err) {
-        if (err.toString().includes("Tried to activate an app that was not installed")) {
+        if (err.toString().includes("AppNotInstalled")) {
           return new HoloError("Failed to sign-in: Agent unknown to this host").toJSON()
```

Comment:
This is still the wrong outcome right? The agent isn't unknown. `hc-state -a` shows that the App IS installed.

Reply:
https://github.com/holochain/holochain/blob/7c6a863454b0553b6aaacd4ec63bb9ad84d2264a/crates/holochain/src/conductor/conductor.rs#L598

If an app is not deactive Holochain assumes that it's not installed. It's just a bad error message on Holochain's part.

Reply:
Oh wait. You're right. One sec

Reply:
Fixed